tests/devices/test_emos_ip300_camera.py

```diff
@@ -0,0 +1,107 @@
+"""Tests for the EMOS IP-300 camera"""
```

Comment:
Tests are not needed for devices that do not introduce new features. Given the number of devices supported now, the tests would overload the free tier on Github if we had them for every device.